osbs/conf.py

```diff
@@ -266,12 +266,23 @@ def get_registry_secrets(self):
         else:
             return []
 
-    def get_registry_api_versions(self):
+    def get_registry_api_versions(self, platform=None):
         value = self._get_value("registry_api_versions",
                                 self.conf_section,
                                 "registry_api_versions",
                                 default='v1,v2')
-        return value.split(',')
+        if not platform:
+            return value.split(',')
+        else:
+            section = 'platform:{0}'.format(platform)
+            enable_v1 = self._get_value("enable_v1", section, "enable_v1", False, True)
```

Comment:
Can we explicitly name the keyword arguments here to save having to look around for the method signature? e.g. `default=False, is_bool_val=True`.